docs/ProjectUsage.md

```diff
@@ -1,30 +1,34 @@
-# Usecase
+---
+layout: default
+title: Why VulnerableApp
+nav_order: 5
+---
+# Why VulnerableApp
 
 There are many tools for finding Web Application Vulnerability like [ZAP](https://github.com/zaproxy), Burp etc but while contributing to these Open Source Tools 
 it is very tough to check if the Payloads added or Scripts added are working as expected.
 
-## Solutions
+## What used to be the solution
 ``` 
-1. Generally developers are writing vulnerable applications for testing Payloads before 
-contributing to these tools or
-2. Writing Unit test cases
+1. Tool developers had to manually write applications that are prone to separate security flaws that the tool
+they are developing is able to flag.
+2. Slog it out with unit tests
 ```
-Both the above approaches are having flaws like 
+### Why would the above not work 
 ```
-Approach 1: One developer's vulnerable application can be reused by other developers and 
-as everyone is writing there own applications it is like doing rework again and also there might be chances of various bugs in those applications
-which are not reviewed by anyone
+Approach 1: Tool developers have to focus on ways to crash test their apps , which is cumbersome as each might have to
+develop their own versions of a flawed web-app , debugging those apps might quickly become overwhelming 
 
 Approach 2: Simulating every scenario using unit tests are not feasible.
 ```
-
-Solutions provided by [VulnerableApp](https://github.com/SasanLabs/VulnerableApp):
+## What we propose
+#### The [VulnerableApp](https://github.com/SasanLabs/VulnerableApp):
 ```
 1. Simple SpringBoot Application which is very easy to run.
 2. Simple Annotation based and URL Path based vulnerable code execution
 3. Easy integration with any of the Web Application Vulnerability Finding Tools.
 4. Scenarios included are very common and exploitable using any Vulnerability finding tools.

```

Comment:
can we please write your wording, some pointers are :
Simple, Easier integration, Extensible as easier to add new vulnerabilities, simple UI automatically build with some small html snippets, database of whole lot of vulnerabilities, utilized across other scanner applications, ZAP's point etc.

Reply:
This is done too , 
reworded the section